index.d.ts

```diff
@@ -68,7 +71,7 @@ declare class Block {
      * Each bounding box is an array of the form [xmin, ymin, zmin, xmax, ymax, zmax].
      * Depends on the type and state of the block.
      */
-    shapes: Array<Array<number>>
+    shapes: Shape[][]
```

Comment:
You put the array in Shape so it's only one array now

Shape[]

Reply:
oof, I should really stop doing stuff late night. Thank you!